wasm/tests/test.c

```diff
@@ -1193,6 +1193,12 @@ void test_opa_json_dump()
     opa_array_append(opa_cast_array(terminators), opa_null());
 
     test("bool/null terminators", opa_strcmp(opa_json_dump(terminators), "[true,false,null]") == 0);
+
+    opa_value *non_string_keys = opa_object();
+    opa_array_t *arrk = opa_cast_array(opa_array());
+    opa_array_append(arrk, opa_number_int(1));
+    opa_object_insert(opa_cast_object(non_string_keys), &arrk->hdr, opa_number_int(1));
+    test("objects/non string keys", opa_strcmp(opa_json_dump(non_string_keys), "{\"[1]\":1}") == 0)
```

Comment:
nit: We've got a `test_str_eq` helper for this sort of thing now that gives better error messages on failures

Reply:
Fixed!